src/ape_accounts/_cli.py

```diff
@@ -64,22 +64,36 @@ def _list(cli_ctx, show_all_plugins):
             click.echo()
 
 
-@cli.command(short_help="Create a new keyfile account with a random private key")
+@cli.command(short_help="Create a new keyfile account with a random mnemonic seed phrase")
+@click.option(
+    "--word-count",
+    "word_count",
+    help="Number of words to use to generate seed phrase",
+    default=12,
+    show_default=True,
+)
+@click.option(
+    "--hdpath",
+    "custom_hd_path",
+    help="Specify an HD path for deriving seed phrase",
+    default="m/44'/60'/0'/0/0",
+    show_default=True,
+)
 @non_existing_alias_argument()
 @ape_cli_context()
-def generate(cli_ctx, alias):
+def generate(cli_ctx, alias, hide_mnemonic, word_count, custom_hd_path):
     path = _get_container().data_folder.joinpath(f"{alias}.json")
-    extra_entropy = click.prompt(
-        "Add extra entropy for key generation...",
-        hide_input=True,
-    )
```

Comment:
Add this back:

```py
    click.prompt(
        "Add extra entropy for key generation...",
        hide_input=True,
    )
```

Reply:
If I'm understanding correctly, extra entropy is not used in `EthAccount.create_with_mnemonic` and therefore isn't needed if we only have the mnemonic path? See here for details: https://eth-account.readthedocs.io/en/stable/eth_account.html

Reply:
`os.urandom` (which is used internally for this method) requries a certain amount of entropy before it completes. So, having the user type in a bunch of random stuff actually helps `urandom` improve it's randomness output, even if we don't pass it anywhere

feel free to add this as a comment above this line

Reply:
ok great sounds good thanks!

Reply:
this should be complete now